### PR TITLE
[cypress] + com_checkin

### PR DESCRIPTION
**Pull request by alikon**
### Summary of Changes
add a simple test for the view `/administrator/index.php?option=com_checkin`

### Testing Instructions
run
`npm run cypress:run -- --spec tests/System/integration/administrator/components/com_checkin/Default.cy.js` 

### Actual result BEFORE applying this Pull Request
N/A

### Expected result AFTER applying this Pull Request
a test for the view

### Link to documentations
Please select:
- [ ] Documentation link for docs.joomla.org: <link>
- [ ] No documentation changes for docs.joomla.org needed

- [ ] Pull Request link for manual.joomla.org: <link>
- [ ] No documentation changes for manual.joomla.org needed

### CODE REVIEW

File: tests/System/integration/administrator/components/com_checkin/Default.cy.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  it('have a title', () => {
+    cy.get('h1.page-title').should('contain.text', 'Maintenance: Global Check-in');
+  });
+
+  it('can display message', () => {
+    cy.get('p.lead').should('contain.text', 'There are no tables with checked out items');
+  });
+
+  it('can display a list of checked out', () => {
+    cy.db_createArticle({ title: 'Test article' }).then((article) => {
```
Comment (laoneo) [inline tests/System/integration/administrator/components/com_checkin/Default.cy.js]: As you are testing the checkin feature, you can create here the article with the "checked_out" and "checked_out_time" attributes. Then you don't need to open an article and save it.

It speeds up the test and it actually tests only what you want to test and not also article saving. Same can be applied below.

Comment (alikon) [reply tests/System/integration/administrator/components/com_checkin/Default.cy.js]: ah ok

**Comment by muhme** [comment]
✅ successfully tested this additional test spec on the branches dev-4.4, dev-5.1, dev-5.2 and dev-6.0, as single spec and within the full System Test

File: tests/System/integration/administrator/components/com_checkin/Default.cy.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
+describe('Test in backend that the checkin', () => {
+  beforeEach(() => {
+    cy.doAdministratorLogin();
+    cy.visit('/administrator/index.php?option=com_checkin');
+  });
+
+  it('have a title', () => {
```
Comment (brianteeman) [inline tests/System/integration/administrator/components/com_checkin/Default.cy.js]:
```suggestion
  it('has a title', () => {
```

File: tests/System/integration/administrator/components/com_checkin/Default.cy.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+  });
+
+  it('can display a list of checked out', () => {
+    cy.db_createArticle({ title: 'Test article', checked_out: '1', checked_out_time: '2024-01-01 20:00:00' }).then(() => {
+      cy.visit('/administrator/index.php?option=com_checkin');
+      cy.get('tr.row0').should('contain.text', 'content');
+    });
+  });
+
+  it('can checkin items', () => {
```
Comment (brianteeman) [inline tests/System/integration/administrator/components/com_checkin/Default.cy.js]:
```suggestion
  it('can check in items', () => {
```

ref https://manual.joomla.org/docs/user-interface-text/action_or_description

**Comment by laoneo** [comment]
Thanks!

**Pull request merged**